src/app/pages/questions/components/question/text-input/text-input.component.ts

```diff
@@ -53,13 +53,13 @@ export class TextInputComponent implements OnInit {
     ampm: 'AM/PM'
   }
   textValue = ''
-
   value = {}
+  DEFAULT_DATE_FORMAT = 'DD/MM/YYYY'
```

Comment:
I think this is enough for now, but would be good to do this via remote config (or protocol). Please create an issue so it can be handled later.